### PR TITLE
write tips n tricks

### PR DESCRIPTION
**Pull request by StefanFitoussi**
Please review my code :)

### CODE REVIEW

**Review by HendrikNeuschulz — CHANGES_REQUESTED** [review]
its looks fantastic but ..

could u change the headline to another one?

**Review by MeenaFoerderer — CHANGES_REQUESTED** [review]
Plz change headline :D

**Comment by StefanFitoussi** [comment]
Okay :)

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
 # This is a headline
+
+This in not a headline ;)
```
Comment (HendrikNeuschulz) [inline README.md:3]: why not?

**Pull request merged**